src/data/store/selectors/nodeSelection.selectors.ts

```diff
@@ -27,7 +28,8 @@ export const getSelectedNodeList = (temporalEpoch: number) => (state: RootState)
 
 export const getOrderedSelectedNodeList = (temporalEpoch: number) => (state: RootState) =>
     (state.nodeSelection.selectedNodeList[temporalEpoch] ?? [])
-        .map((id) => state.nodeSelection.nodeList[temporalEpoch][id])
-        .toReversed();
+        .map((id) => state.nodeSelection.nodeList[temporalEpoch]?.[id])
+        .filter((node) => Boolean(node))
```

Comment:
`node` -> `nodeState` 
also checking for `nodeState === undefined` is more readable